netkan/pyproject.toml

```diff
@@ -0,0 +1,112 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "netkan_indexer"
+version = "1.0"
+dependencies = [
+    "boto3",
+    "click",
+    "gitpython",
+    "pynamodb",
+    # 2019-11-01 capping to 2.8.0 - https://github.com/boto/botocore/commit/e87e7a745fd972815b235a9ee685232745aa94f9
+    "python-dateutil>=2.1,<2.8.1",
```

Comment:
This timestamp is getting a little long in the tooth (almost 5 years. Any idea whether we still need this version cap?

Reply:
I am not sure, but I wanted this PR to purely updating python and changing to pyproject.toml. There will be follow up work, but I prefer those changes to be separate.

Reply:
That's fair.